monitoring/uss_qualifier/scd/executor/test_steps/inject_flight.py

```diff
@@ -18,6 +19,15 @@
 def execute(runner, step: TestStep, step_ref: TestStepReference, target: TestTarget) -> None:
     print("[SCD]     Step: Inject flight {} to {}".format(step.inject_flight.name, target.name))
     try:
+        reference_time = datetime.utcnow() + timedelta(seconds=pytimeparse.parse(step.inject_flight.planning_time))
```

Comment:
Small note: because planning_time is a StringBasedTimeDelta, the timedelta value can be read simply as [`step.inject_flight.planning_time.timedelta`](https://github.com/interuss/dss/blob/9f6a372d1ada89873361872c9f37fcc9e6bab175/monitoring/monitorlib/typing.py#L222) rather than `timedelta(seconds=pytimeparse.parse(step.inject_flight.planning_time))`